plugin/src/main/groovy/com/btkelly/gnag/utils/ProjectHelper.groovy

```diff
@@ -20,18 +20,31 @@ import org.gradle.api.Project
 /**
  * Created by bobbake4 on 4/19/16.
  */
-class ReportHelper {
+class ProjectHelper {
 
-    private final Project project;
+    private final Project project
 
-    ReportHelper(Project project) {
+    ProjectHelper(Project project) {
         this.project = project
     }
 
-    // todo: this doesn't seem like it belongs in the ReportHelper. refactor?
-    public List<File> getAndroidSources() {
-        project.android.sourceSets.inject([]) {
-            dirs, sourceSet -> dirs + sourceSet.java.srcDirs
+    public Project getProject() {
+        return project
+    }
+
+    public boolean isAndroidProject() {
+        return project.hasProperty("android")
```

Comment:
Yeah that's nice; hopefully no-one else is squatting the `android` extension name. Is uniqueness enforced there, do you know? I can't imagine how it could be... maybe you have to qualify if two plugins add an extension with the same name?